<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestImageCanvasMainDialog</class>
 <widget class="QDialog" name="QtTestImageCanvasMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>585</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #808080, stop: 1 #ffffff);
}
QPushButton { background-color: rgb(255, 255, 255);  }
QPushButton { color: rgb(0,0,0); }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layout_image">
     <item>
      <widget class="QPushButton" name="button_image">
       <property name="text">
        <string>Put image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_image_n_cols">
       <property name="text">
        <string>(n_cols =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_n_cols">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_image_close">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_image">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_color_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <property name="text">
        <string>Color system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="box_color_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>invert</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_coordinat_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <property name="text">
        <string>Coordinat system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="box_coordinat_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>screen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>graph</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
